src/locale/zh_Hant/LC_MESSAGES/django.po

```diff
@@ -4142,6 +4142,10 @@ msgstr "提供的圖片過小（{width}\\u00d7{height} 像素）。"
 msgid "The image you provided is not quadrate."
 msgstr "提供的圖片並非方型。"
 
+#: users/forms.py:
```

Comment:
nit: Please also add this message to `src/locale/en-US/LC_MESSAGES/django.po`.

Reply:
Hello @mattwang44, thanks for your remind. I've added it. Please have a check.

Reply:
NP. Thanks for the efforts!